composer.json

```diff
@@ -9,7 +9,8 @@
     "config": {
         "allow-plugins": {
             "dealerdirect/phpcodesniffer-composer-installer": true,
-            "magento/*": true
+            "magento/*": true,
+            "php-http/discovery": true
```

Comment:
Could you please explain why it was added?

Reply:
hi @ihor-sviziev 

it will auto install PSR, here's for the documentation of this https://packagist.org/packages/php-http/discovery